internal/pkg/template/templates/environment/partials/s3-elb-access-logs.yml

```diff
@@ -0,0 +1,39 @@
+ELBAccessLogsBucketPolicy:
```

Comment:
nit: Would you mind renaming the file to just `elb-access-logs.yml` since S3 is the only possible output location